src/lib/models/Bid.js

```diff
@@ -19,6 +19,24 @@ class Bid extends Model {
   static findInBidGroup(bidGroupId, bidIndex) {
     return this.findOne({ bidGroupId, bidIndex })
   }
+
+  static count() {
+    return this.db
+      .query(`SELECT COUNT(*) FROM ${Bid.tableName}`)
+      .then(r => r[0].count)
+  }
+
+  static popular(limit) {
```

Comment:
Nitpicking but I like methods that have a verb indicating what they're doing. Like `getPopular` in this case.
`Bid.popular` could mean "make it popular" as well as "get the popular"

Reply:
Done, thank you!